doc/source/train/getting-started-pytorch-lightning.rst

```diff
@@ -7,9 +7,9 @@ This tutorial walks through the process of converting an existing PyTorch Lightn
 
 Learn how to:
 
-1. Configure the Lightning Trainer so that it runs distributed with Ray and on the correct CPU or GPU device.
+1. Configure the Lightning Trainer so that it runs distributed with Ray and on the correct CPU, GPU or other accelerator devices.
```

Comment:
```suggestion
1. Configure the Lightning Trainer so that it runs distributed with Ray and on the correct CPU, GPU, or other accelerator devices.
```